ogc_ws/src/despatcher/scripts/missionserver.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import os
+import rospy
+from pathlib import Path
+import paramiko
+import time
+
+serverfolder = "/home/ubuntu/Waypoints/"
+
+class MissionServer():
+	def __init__(self):
+		home_dir = str(Path.home())
+		server_user = "ubuntu"
+		server_ip = "46.137.242.90"
```

Comment:
We'll keep this first, but eventually the following must be done:

1. The server ip must be broken out into the identifiers file (and I'll sync it with the way sbd handles it)
2. Remove the private key and use public key authorization